view/frontend/web/css/source/_module.less

```diff
@@ -68,6 +68,10 @@
                     margin-left: 5px;
                 }
 
+                .rvvup-card-img {
+                    width: auto !important;
```

Comment:
**❔ question:** Why is this required for cards specifically?

Reply:
Otherwise, it won't be displayed for a full size.
We have other specific one for rvvup-yapily-img

Reply:
Tested this out to see how it looks:
![image](https://github.com/rvvup/magento-plugin/assets/96258159/9a72290f-3321-40da-806c-188b305b6b66)